### PR TITLE
Specified shell in Makefile and minor compatability change

### PR DESCRIPTION
**Pull request by arkocal**
Makefile now uses (shell pwd) instead of {pwd}. Please test whether this works as intended on your system as well before merge.

The important changes are at lines 28 and 57, my editor has automatically corrected some trailing whitespaces and space/tab mixes, so the commit looks a bit busier than it actually is.

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -48,34 +50,34 @@ endif
 	fi;
 	@touch $@
 
-build: .init 
-	@$(call msg,"Building IoT connector ..."); 
+build: .init
+	@$(call msg,"Building IoT connector ...");
 	@/bin/bash -c "./docker.sh create "
```
Comment (nevets963) [inline Makefile]: @arkocal If you're specifying the location of bash, we can also use the constant here?

Comment (arkocal) [reply Makefile]: Yes, we can.
I think specifying the SHELL variable is a better choice overall, as this is also recognized by make and used for commands which are not passed to a shell as a parameter (like in lines 33 to 37 or 32-33). For stability it is preferable to letting make ue the system default, which can be different on different systems can cause ugly bugs which mostly are not easily visible.

Comment (nevets963) [reply Makefile]: @arkocal Sounds good, would you mind updating this PR and adding this? Thanks, Steve

**Comment by arkocal** [comment]
@nevets963 hi Steven, I have removed those lines and tested on my system. Please check whether it works on your machine as well when you have the time.

**Comment by nevets963** [comment]
@arkocal The green debug messages now have newline (\n) and tab (\t) displaying and not taking affect. Can you look at this?

**Comment by arkocal** [comment]
Looks like bash needs the -e argument to echo for escape sequences to take effect.

**Comment by nevets963** [comment]
@arkocal It works, nice one!

**Pull request merged**